CHANGELOG.md

```diff
@@ -1,3 +1,36 @@
+## v1.7.0 (02/11/2020)
+
+**olp-sdk-core**
+
+- Added `sentWith` query parameter for each request.
+- Updated dependencies.
+
+**olp-sdk-authentication**
+
+- Added `sentWith` query parameter for each request.
+- Updated dependencies.
+
+**olp-sdk-fetch**
+
+- Updated dependencies.
+
+**olp-sdk-dataservice-api**
+
+- Added generated functions and interfaces for supporting Authorization APIs. You can find them in `@here/olp-sdk-dataservice-api/lib/authorization-api-v1.1.ts`.
+- Added Content-Encoding header to the PUT blob request.
+- **Breaking change** Removed deprecated items with the deprecated time more than 6 months. @see https://github.com/heremaps/here-data-sdk-typescript/pull/385/
+- Updated dependencies.
+
+**olp-sdk-dataservice-read**
+
+- Deprecated using version from request class in `MetadataCacheRepository`.
```

Comment:
Using version in `MetadataCacheRepository` request class is now deprecated and will be removed by MM.YYYY.